app/models/miq_request_workflow.rb

```diff
@@ -113,6 +113,7 @@ def init_from_dialog(init_values)
             init_values[field_name] = [val, field_values[:values][val]]
           else
             field_values[:values].each do |tz|
+              next unless tz.kind_of?(Array)
```

Comment:
Curious why this was added in this PR?  Did something change between HashStruct and OpenStruct that warranted this one?

Reply:
Possibly a bad rebase?  The commit I rebased was super old, and also between a lot of commits that already go merged (and some that didn't) from this collection of improvments:

https://github.com/ManageIQ/manageiq/compare/master...NickLaMuro:multiple_miq_provision_workflow_query_optimizations

Entirely possible that I goofed something, so I will revisit.

Reply:
@Fryguy scratch that, I explained it in the commit message.